### PR TITLE
feat: multi-container input and output [PAGOPA-1542] [PAGOPA-1543]

### PR DESCRIPTION
**Pull request by AngeloCaporaso**
#### List of Changes

- `@BlobTrigger` was replaced with `@QueueueTrigger`: the queue contains the `Blob.Created` events published to System Topic via `EventGrid`;
- partition key creation logic for the `DebtPositionsTable` was added: the key is composed of the concatenation of the container (one for the corporate) and the name of the file loaded by the corporate; this change was necessary because the name of the blob (or file) is no longer unique;
- the logic of result uploading (output or error) into the correct container, i.e., the one corresponding to the corporate that uploaded the `PaymentNotice`, has been updated;

#### Motivation and Context

- These changes are necessary to move from a single container with heterogeneous corporate files to a multiple container in which each corporate has a dedicated container; now storage is partitioned by corporate.

#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

File: src/main/java/it/gov/pagopa/canoneunico/functions/CuCreateDebtPosition.java [hunk 1]
```diff
@@ -175,12 +175,13 @@ private int createDebtPosition(Logger logger, DebtPositionRowMessage row, String
                 .fullName(row.getDebtorName())
                 .email(row.getDebtorEmail())
                 .companyName(row.getCompanyName())
+                .switchToExpired(false)
```
[the remark this answers is not in the document]
Comment (AngeloCaporaso) [reply src/main/java/it/gov/pagopa/canoneunico/functions/CuCreateDebtPosition.java:178]: it is a [required](https://github.com/pagopa/pagopa-debt-position/blob/fb5c0b75fcf5b927830168e3cd13eabe9a651518/src/main/java/it/gov/pagopa/debtposition/model/pd/PaymentPositionModel.java#L53) data for debt-position
changed [here](https://github.com/pagopa/pagopa-debt-position/pull/198)

File: src/main/java/it/gov/pagopa/canoneunico/functions/CuCsvParsing.java [hunk 2]
```diff
@@ -27,47 +34,94 @@ public class CuCsvParsing {
...
         }
+        EventGridEvent event = eventGridEvents.get(0);
+        if (!event.getEventType().equals("Microsoft.Storage.BlobCreated")) {
+            throw new CanoneUnicoException("[CuCsvParsing] Event not equals to Microsoft.Storage.BlobCreated.");
+        }
+
+        logger.log(Level.INFO, () -> String.format("[id=%s][CuCsvParsing] Call event type %s handler.", context.getInvocationId(), event.getEventType()));
+        StorageBlobCreatedEventData blobData = event.getData().toObject(StorageBlobCreatedEventData.class, new DefaultJsonSerializer());
 
+        if (blobData.getContentLength() > 1e+8 || blobData.getContentLength() == 0) { // if file greater than 100 MB or 0 MB
```
[the remark this answers is not in the document]
Comment (AngeloCaporaso) [reply src/main/java/it/gov/pagopa/canoneunico/functions/CuCsvParsing.java:108]: to prevent OutOfMemoryException like [this](https://learn.microsoft.com/en-us/answers/questions/643009/azure-function-outofmemoryexception-after-100-mb-a), test this corner case is required

[the remark this answers is not in the document]
Comment (pasqualespica) [reply src/main/java/it/gov/pagopa/canoneunico/functions/CuCsvParsing.java:108]: ok, add ref comment before comment

**Review by pasqualespica — APPROVED** [review]
great work 🚀

**Review by pasqualespica — APPROVED** [review]
🚀
